src/collective/upload/tests/test_controlpanel.py

```diff
@@ -60,28 +59,42 @@ class RegistryTestCase(unittest.TestCase):
 
     def setUp(self):
         self.portal = self.layer['portal']
-        self.registry = getUtility(IRegistry)
-        self.settings = self.registry.forInterface(IUploadSettings)
         setRoles(self.portal, TEST_USER_ID, ['Manager'])
 
+    def _get_setting(self, setting):
```

Comment:
-1 also on all of these changes in these module for the same reasons expressed above.